README.md

```diff
@@ -24,7 +24,23 @@ pip install pytket-quantinuum
 This will install `pytket` if it isn't already installed, and add new classes
 and methods into the `pytket.extensions` namespace.
 
-The [example notebooks](https://github.com/CQCL/pytket-quantinuum/tree/develop/examples) also demonstrate some of the key features of pytket-quantinuum.
+## Examples
+
+For example usage of features, see the [pytket-quantinuum section](https://docs.quantinuum.com/h-series/trainings/getting_started/pytket_quantinuum/pytket_quantinuum.html) of the documentation website.
```

Comment:
Maybe I should link to specific subsections from [this page](https://docs.quantinuum.com/h-series/trainings/getting_started/pytket_quantinuum/pytket_quantinuum.html) as well as the knowledge articles? However that would lead to a lot of links in the README.

Reply:
Will defer to @cqc-alec, but I think easier for only one link to exist from H-Series to the TKET page.

Reply:
I'll leave it as is for now